chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchBasicDetails.java

```diff
@@ -0,0 +1,752 @@
+package gov.healthit.chpl.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonProperty.Access;
+import gov.healthit.chpl.domain.surveillance.Surveillance;
+import gov.healthit.chpl.util.Util;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlTransient;
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ * Certified Product Search Basic Details entity.
+ */
+@XmlType(namespace = "http://chpl.healthit.gov/listings")
```

Comment:
All these XML-related annotations probably do not need to be in this class. The original CertifiedProductSearchDetails gets written out nightly to an XML file which is why all the annotations are here with details comments, but I don't think this class will need any of that.